ledger/cow_test.go

```diff
@@ -112,7 +113,7 @@ func TestCowBalance(t *testing.T) {
 	accts0 := randomAccounts(20, true)
 	ml := mockLedger{balanceMap: accts0}
 
-	c0 := makeRoundCowState(&ml, bookkeeping.BlockHeader{}, 0, 0)
+	c0 := makeRoundCowState(&ml, bookkeeping.BlockHeader{}, config.ConsensusParams{}, 0, 0)
```

Comment:
please avoid passing this `config.ConsensusParams{}`, even if that works right now.
Just pick the ConsensusParams for the default protocol version.

Reply:
done